### DISCUSSION TITLE
Unable to apply for Github education student pack

### DISCUSSION THREAD

**Discussion by nandineesingla** [community/community#51212] [GitHub Education]
Hi,
Everytime I apply with my ID card my request is rejected with following message -
"You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio."

But my user profile is complete.

I cannot use  my school email as my school doesn't allow emails from outside school domain.

Please help me with this.

**Comment by khalidbelk** [comment]
Had the same issue being rejected immediately 5 times and I didn't understand why. Here are some tips that could help you solve the issue. Looking at your profile I see you're doing the same thing as me and I thought my profile was complete as well.

In your profile settings :
- Set a status (whatever)
- Add a Company
- Add a Location
- Display current local time 
- Add a website URL

And of course, take a pic of the school ID with the device you're applying from (even if it's you PC) that clearly shows all the details (light is very important).

Hope that helps you, don't forget to upvote this and tell me if it worked ;)

**Reply by samus-aran** [reply]
Awesome reply @khalidbelk 👍 

I would also add to your suggestion to log out and log back in again after updating your profile and before re-applying to GitHub Global Campus.

@nandineesingla did @khalidbelk's answer help you? ⭐

**Reply by Sharon0726** [reply]
I just done everything but it still can't work.

**Comment by Hashem-Soud1** [comment]
Did you find a solution my friend because I am facing the same problem؟

**Comment by Lin-Den-Dev** [comment]
I logged in via my phone's browser and just took a pic.

None of the @khalidbelk steps helped me until I've taken a pic of my student's card with a camera.

**Comment by Dhanasty** [comment]
Even I had the same issue for quite a few days. Here are the things I did that helped me solve the issue.

1)  Complete the profile settings
2) Use your mobile phone camera to take a picture of your student id card

**Comment by innomaxx** [comment]
I have the same issue. I completed my profile with name as in passed document and added bio. But I still experience two issues :

Your application cannot be reviewed until you fix the following:

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

All the steps are done. What I am doing wrong?

**Reply by khalidbelk** [reply]
As I can see you didn't set any organization (which could be helpful for the verification process). Also did you sent a clear image of your  student ID with good light that shows all the details ? You can try that out, as you may know I don't work for Github nor I can't guarantee you a 100% result, I'm just here trying to help whoever is facing the same issue that I had. For my case, I had to desperately try at least 5 times before getting a result. Good luck, and if it works don't forget to give an answer

**Reply by innomaxx** [reply]
> As I can see you didn't set any organization (which could be helpful for the verification process). Also did you sent a clear image of your student ID with good light that shows all the details ? You can try that out, as you may know I don't work for Github nor I can't guarantee you a 100% result, I'm just here trying to help whoever is facing the same issue that I had. For my case, I had to desperately try at least 5 times before getting a result. Good luck, and if it works don't forget to give an answer

Thanks for your reply. Okay I will try set an organization. It can be any type or should be related to education (for example I need add my university or it can be any organization or company?)
Sure I will post here when the issue will be resolved

**Reply by khalidbelk** [reply]
It could be your University. And don't forget the the photo is very important (don't take it from too far or without light). If it doesn't work, retry ! Good luck :) !

**Reply by Adolar0042** [reply]
I have the same problem and contacted support. They said you have to change your name to the name on your academic proof and keep it as that permanently.
![image](https://github.com/community/community/assets/39769465/78d1ca71-7de8-47be-a999-cc027a4234c0)

**Reply by khalidbelk** [reply]
> > As I can see you didn't set any organization (which could be helpful for the verification process). Also did you sent a clear image of your student ID with good light that shows all the details ? You can try that out, as you may know I don't work for Github nor I can't guarantee you a 100% result, I'm just here trying to help whoever is facing the same issue that I had. For my case, I had to desperately try at least 5 times before getting a result. Good luck, and if it works don't forget to give an answer
> 
> Thanks for your reply. Okay I will try set an organization. It can be any type or should be related to education (for example I need add my university or it can be any organization or company?) Sure I will post here when the issue will be resolved

Any updates @innomaxx :) ?

**Reply by innomaxx** [reply]
> > > As I can see you didn't set any organization (which could be helpful for the verification process). Also did you sent a clear image of your student ID with good light that shows all the details ? You can try that out, as you may know I don't work for Github nor I can't guarantee you a 100% result, I'm just here trying to help whoever is facing the same issue that I had. For my case, I had to desperately try at least 5 times before getting a result. Good luck, and if it works don't forget to give an answer
> > 
> > 
> > Thanks for your reply. Okay I will try set an organization. It can be any type or should be related to education (for example I need add my university or it can be any organization or company?) Sure I will post here when the issue will be resolved
> 
> Any updates @innomaxx :) ?

No updates for now :(
Deferred the task to time I will have more time.
Thanks for your concern, I will try keep you up to date about my attempts

**Reply by khalidbelk** [reply]
But you're already Pro as I can see on you profile what's the issue ?

**Comment by Nobatgeldi** [comment]
Today I faced with the same problem as you. I changed the name of my profile, it didn't work. After that, I added my surname to my profile with name, it worked. It seems like just a name is not enough.
